bundles/config/org.eclipse.smarthome.config.dispatch/src/main/java/org/eclipse/smarthome/config/dispatch/internal/ConfigDispatcher.java

```diff
@@ -69,9 +66,23 @@
  *
  * @author Kai Kreuzer - Initial contribution and API
  * @author Petar Valchev - Added sort by modification time, when configuration files are read
+ * @author Ana Dimova - reduce to a single watch thread for all class instances
  */
 public class ConfigDispatcher extends AbstractWatchService {
 
+    private static String getPathToWatch() {
+        String progArg = System.getProperty(SERVICEDIR_PROG_ARGUMENT);

```

Comment:
Err, sorry, forget my comment - I actually didn't notice that this is within the ConfigDispatcher class, I somehow thought it was related to automation... Never mind!